src/components/sidebar/ShipsContainer.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { defaultShip, PlayerShipsContext } from '../../context/PlayerShipsContext';
+import Ship from './Ship';
+
+const ShipsContainer: React.FC = function () {
+    const ships = useContext(PlayerShipsContext);
+
+    const carrier = ships?.Carrier || defaultShip;
+    const battleShip = ships?.BattleShip || defaultShip;
+    const cruiser = ships?.Cruiser || defaultShip;
+    const submarine = ships?.Submarine || defaultShip;
+    const destroyer = ships?.Destroyer || defaultShip;
+
+    // Maybe just make the above ^ into an array??
+    const playerShips = [carrier, battleShip, cruiser, submarine, destroyer];
+
+    const shipsMap = playerShips.map((ship) => {
```

Comment:
I think this could use a better name to describe the behavior of displaying ship info.

Reply:
Renamed shipsMap to shipsDisplay.